### PR TITLE
Removed name filter

### PR DESCRIPTION
**Pull request by tzufgoogle**
1, removed name filter
2. added wikidata and wikipedia tags for extraction conditions

### CODE REVIEW

File: cabby/geo/map_processing/map_structure.py [hunk 1]
```diff
@@ -102,15 +102,18 @@ def get_poi(self) -> Tuple[GeoSeries, GeoSeries]:
       (1) The POI that are not roads; and (2) the roads POI.
     '''
 
-    tags = {'name': True, 'building': True, 'amenity': True}
+    tags = {'name': True,
+        'building': True,
```
Comment (jpalowitch) [inline cabby/geo/map_processing/map_structure.py:106]: nit: move first dict element down a line, or indent other elements to first element

File: cabby/geo/walk.py [hunk 2]
```diff
@@ -82,9 +82,11 @@ def get_end_poi(map: map_structure.Map) -> Optional[GeoSeries]:
   Returns:
     A single POI.
   '''
+  # Filter with name.
+  named_poi = map.poi[map.poi['name'].notnull()]
 
   # Filter large POI.
-  small_poi = map.poi[map.poi['s2cellids'].str.len() <= 4]
+  small_poi = named_poi[named_poi['s2cellids'].str.len() <= 4]
```
Comment (jpalowitch) [inline cabby/geo/walk.py:89]: Please define `4` as module-level variable

File: cabby/geo/map_processing/map_structure.py (hunk 1 above)
Comment (tzufgoogle) [reply cabby/geo/map_processing/map_structure.py]: fixed

File: cabby/geo/walk.py (hunk 2 above)
Comment (tzufgoogle) [reply cabby/geo/walk.py]: Added

**Review by tzufgoogle — COMMENTED** [review]
finished making changes: adding variable and fixed indentation

**Pull request merged**